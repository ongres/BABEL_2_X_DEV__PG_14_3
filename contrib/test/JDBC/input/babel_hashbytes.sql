create table test1 (c1 VARCHAR(32))
GO

insert test1 values ('This is a test.')
GO

insert test1 values ('This is test 2.')
GO

insert test1 values (' ')
GO

insert test1 values ('')
GO

select hashbytes('md2', c1) from test1
GO

select hashbytes('md4', c1) from test1
GO

select hashbytes('md5', c1) from test1
GO

select hashbytes('sha', c1) from test1
GO

select hashbytes('sha1', c1) from test1
GO

select hashbytes('sha2_256', c1) from test1
GO

select hashbytes('sha2_512', c1) from test1
GO

select hashbytes('fake', c1) from test1
GO



create table test2 (c1 VARBINARY(32))
GO

insert test2 values ('1234567890')
GO

insert test2 values (1234567890)
GO

insert test2 values (0)
GO

insert test2 values (1)
GO

select hashbytes('md2', c1) from test2
GO

select hashbytes('md4', c1) from test2
GO

select hashbytes('md5', c1) from test2
GO

select hashbytes('sha', c1) from test2
GO

select hashbytes('sha1', c1) from test2
GO

select hashbytes('sha2_256', c1) from test2
GO

select hashbytes('sha2_512', c1) from test2
GO

select hashbytes('fake', c1) from test2
GO


drop table test1
GO

drop table test2
GO

