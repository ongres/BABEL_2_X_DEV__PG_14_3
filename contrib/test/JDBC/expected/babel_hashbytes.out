create table test1 (c1 VARCHAR(32))
GO

insert test1 values ('This is a test.')
GO
~~ROW COUNT: 1~~


insert test1 values ('This is test 2.')
GO
~~ROW COUNT: 1~~


insert test1 values (' ')
GO
~~ROW COUNT: 1~~


insert test1 values ('')
GO
~~ROW COUNT: 1~~


select hashbytes('md2', c1) from test1
GO
~~START~~
varbinary
<NULL>
<NULL>
<NULL>
<NULL>
~~END~~


select hashbytes('md4', c1) from test1
GO
~~START~~
varbinary
<NULL>
<NULL>
<NULL>
<NULL>
~~END~~


select hashbytes('md5', c1) from test1
GO
~~START~~
varbinary
120EA8A25E5D487BF68B5F7096440019
2A70956B63937422ED1113DAFA9D31BA
7215EE9C7D9DC229D2921A40E899EC5F
D41D8CD98F00B204E9800998ECF8427E
~~END~~


select hashbytes('sha', c1) from test1
GO
~~START~~
varbinary
AFA6C8B3A2FAE95785DC7D9685A57835D703AC88
9F2E1AFD2DEF81E37FCDC48DA0DB65D8A2EC232E
B858CB282617FB0956D960215C8E84D1CCF909C6
DA39A3EE5E6B4B0D3255BFEF95601890AFD80709
~~END~~


select hashbytes('sha1', c1) from test1
GO
~~START~~
varbinary
AFA6C8B3A2FAE95785DC7D9685A57835D703AC88
9F2E1AFD2DEF81E37FCDC48DA0DB65D8A2EC232E
B858CB282617FB0956D960215C8E84D1CCF909C6
DA39A3EE5E6B4B0D3255BFEF95601890AFD80709
~~END~~


select hashbytes('sha2_256', c1) from test1
GO
~~START~~
varbinary
A8A2F6EBE286697C527EB35A58B5539532E9B3AE3B64D4EB0A46FB657B41562C
48EF5D298A862B6F74338EBB5281F5212D8221A2FD8CCE827BE72779BFFD25DD
36A9E7F1C95B82FFB99743E0C5C4CE95D83C9A430AAC59F84EF3CBFAB6145068
E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855
~~END~~


select hashbytes('sha2_512', c1) from test1
GO
~~START~~
varbinary
F3BF9AA70169E4AB5339F20758986538FE6C96D7BE3D184A036CDE8161105FCF53516428FA096AC56247BB88085B0587D5EC8E56A6807B1AF351305B2103D74B
C56E97A46365E4937107B7E8E21CF2377746137935522EBE7A875BF38DC1044603D72FBF69401B63927902A05AC040455F7680225A2731BF9DB1DF5533F26726
F90DDD77E400DFE6A3FCF479B00B1EE29E7015C5BB8CD70F5F15B4886CC339275FF553FC8A053F8DDC7324F45168CFFAF81F8C3AC93996F6536EEF38E5E40768
CF83E1357EEFB8BDF1542850D66D8007D620E4050B5715DC83F4A921D36CE9CE47D0D13C5D85F2B0FF8318D2877EEC2F63B931BD47417A81A538327AF927DA3E
~~END~~


select hashbytes('fake', c1) from test1
GO
~~START~~
varbinary
<NULL>
<NULL>
<NULL>
<NULL>
~~END~~




create table test2 (c1 VARBINARY(32))
GO

insert test2 values ('1234567890')
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: cannot coerce string literal to varbinary datatype)~~


insert test2 values (1234567890)
GO
~~ROW COUNT: 1~~


insert test2 values (0)
GO
~~ROW COUNT: 1~~


insert test2 values (1)
GO
~~ROW COUNT: 1~~


select hashbytes('md2', c1) from test2
GO
~~START~~
varbinary
<NULL>
<NULL>
<NULL>
~~END~~


select hashbytes('md4', c1) from test2
GO
~~START~~
varbinary
<NULL>
<NULL>
<NULL>
~~END~~


select hashbytes('md5', c1) from test2
GO
~~START~~
varbinary
67CA34A303095D664C1442075B82F4DB
F1D3FF8443297732862DF21DC4E57262
F1450306517624A57EAFBBF8ED995985
~~END~~


select hashbytes('sha', c1) from test2
GO
~~START~~
varbinary
446F5DA85FD59A1C926CF24CD11E65768321DCA8
9069CA78E7450A285173431B3E52C5C25299E473
479E04F3D12D112B5C04C9EE67E4B1E6E201EA4E
~~END~~


select hashbytes('sha1', c1) from test2
GO
~~START~~
varbinary
446F5DA85FD59A1C926CF24CD11E65768321DCA8
9069CA78E7450A285173431B3E52C5C25299E473
479E04F3D12D112B5C04C9EE67E4B1E6E201EA4E
~~END~~


select hashbytes('sha2_256', c1) from test2
GO
~~START~~
varbinary
1785EC310767F81A8D9FD076D39074261C13EA788B9311DEE3CAFF2ECF00670D
DF3F619804A92FDB4057192DC43DD748EA778ADC52BC498CE80524C014B81119
B40711A88C7039756FB8A73827EABE2C0FE5A0346CA7E0A104ADC0FC764F528D
~~END~~


select hashbytes('sha2_512', c1) from test2
GO
~~START~~
varbinary
81FDE5A703D85AF347DCE9086679C830533873E5AB2A4B6178891254AE5E8A1ACBE4175AFC801D727372490227F7F6841DAA410E7D43DC90F18E8651B2CABAA1
EC2D57691D9B2D40182AC565032054B7D784BA96B18BCB5BE0BB4E70E3FB041EFF582C8AF66EE50256539F2181D7F9E53627C0189DA7E75A4D5EF10EA93B20B3
57C365278E08F99674DD24F08425B17C71B0511DEA3B5FFA474DEEB26D64CB993EBDA4650583B29CBA6307D7F4DBB42CA11B093DE2B8ECAB16FF52445401FACF
~~END~~


select hashbytes('fake', c1) from test2
GO
~~START~~
varbinary
<NULL>
<NULL>
<NULL>
~~END~~



drop table test1
GO

drop table test2
GO

